Marlin/Marlin.h

```diff
@@ -130,38 +154,63 @@ extern const char axis_codes[XYZE];
 
   #if E_STEPPERS > 1 && HAS_E1_ENABLE
     #define  enable_E1() E1_ENABLE_WRITE( E_ENABLE_ON)
-    #define disable_E1() E1_ENABLE_WRITE(!E_ENABLE_ON)
+    #if ENABLED(HANGPRINTER) && EXTRUDERS == 1
+      #define disable_E1() NOOP
+    #else
+      #define disable_E1() E1_ENABLE_WRITE(!E_ENABLE_ON)
+    #endif
```

Comment:
@tobbelobb — Since `E_STEPPERS` is the same as `EXTRUDERS` and this block begins with `#if E_STEPPERS > 1 && HAS_E1_ENABLE` there's no possibility for `#if ENABLED(HANGPRINTER) && EXTRUDERS == 1` to evaluate as `true`. So `disable_E1` will always be defined as `E1_ENABLE_WRITE(!E_ENABLE_ON)` regardless.

Reply:
Applied a correction where these are undefined / redefined just below…

Reply:
You're right, good catch! I think the correction commit implements the desired behavior properly.